.docker/localstack/s3.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -x
+
+S3=(
```

Comment:
do we use the localstack s3 for anything? I don't think we can mock the aws synthetic that would call s3 in any way, so unclear we could use this (unless we want to confirm aws s3 cp works, but that seems odd to test here).